src/cloudsc_c/dwarf_cloudsc.c

```diff
@@ -33,6 +33,9 @@ int main( int argc, char *argv[] ) {
     omp_threads = atoi( argv[1] );
     ngptot      = atoi( argv[2] );
     nproma      = atoi( argv[3] );
+    if (omp_threads <= 0) {
+      omp_threads = omp_get_max_threads();
+    }
```

Comment:
Can we guard this with an `#ifdef _OPENMP` to retain the ability of building without OpenMP?

Reply:
resolved in next commit